targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_WIO_3G/PinNames.h

```diff
@@ -215,14 +218,14 @@ typedef enum {
     RESET_MODULE = PD_5,
 
     // Generic signals namings
-    LED1        = PB_11,  // LD1 = GREEN
-    LED2        = PB_12,  // Blue
-    LED3        = PB_13, // Red
-    LED4        = PB_14,
-    LED_RED     = LED2,
-    USER_BUTTON = PC_13,
+    LED1        = NC,
```

Comment:
Why are these being changed to NC? Can you add this to the tcommit message?

Why and how are this being modified, how this fixes the bug?

Reply:
The LED on Wio 3G is not traditional LED, so LED1-4 doesn't have any meaning. However, we need to have definition of LED1-4. This is the reason why we tried to change like this. ( Finally, we changed approach for this issue. )